code/code/misc/create_rooms.cc

```diff
@@ -11,8 +11,7 @@
  *****************************************************************************/
 
 #include <stdio.h>
-
-#include <boost/algorithm/string.hpp>
+#include <boost/algorithm/string/predicate.hpp>
```

Comment:
This change just resolves a deprecation warning I was seeing by using a more specific header file to provide the `iequals` symbol.